pkg/common-controller/snapshot_create_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var timeNow = time.Now()
+var timeNow = time.Date(2021, time.January, 1, 0, 0, 0, 0, time.Local)
```

Comment:
Was running into unit test issues without this.
```
        -                       CreationTime:                   s"2021-04-01 17:45:55.275503069 -0700 PDT m=+0.021010896",
        +                       CreationTime:                   s"2021-04-01 17:45:55 -0700 PDT",
```

The patching behavior seems to treat timestamps with less precision.

Reply:
hmm, interesting

Reply:
I think the precision is only for unit tests - I just tried on a cluster and the `creationTime` seems to not have the fractional seconds above.

```
  status:
    boundVolumeSnapshotContentName: snapcontent-6ac333b4-25c5-49d9-a55e-5a7cdc8db7f3
    creationTime: "2021-04-02T01:03:11Z"
    readyToUse: true
```

Reply:
Changed this to be `time.Now().Round(time.Second)` instead.